Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,317 @@
+
+On the other hand major bugs, which might be used to compromise data, that is not in direct connection to the triggered bug, occur only a few times a year.
+This can be seen e.g. in the [OpenStack Security Advisories](https://security.openstack.org/ossalist.html), where there were only 3 major bugs found in 2023.
+While these bugs might appear only rarely their consequences are immense.
+They might be the reason for a whole deployment to be compromised or shut down.
+CSPs should be in contact with people triaging and patching such bugs, to be informed early and to be able to update their deployments, before the bug is openly announced.
+
+#### Human Interference
+
+| Failure Scenario | Probability | Consequences | Failsafe Level Coverage |
+|----|-----|----|----|
+| Minor operating error | High | Temporary outage | L1 |
+| Major operating error | Low | Permanent loss of data | L3 |
+| Cyber threat | High | permanent loss or compromise of data on affected Disk and Host | L1 |
+
+Mistakes in maintaining a data center will always happen.
+To reduce the probability of such a mistake, measures are needed to reduce human error, which is more an issue of sociology and psychology instead of computer science.
+On the other side an attack on an infrastructure cannot be avoided by this.
+Instead every deployment needs to be prepared for an attack all the time, e.g. through security updates.
+The severity of Cyber attacks can also vary broadly: from denial-of-service attacks, which should only be a temporary issue, up until coordinated attacks to steal or destroy data, which could also affect a whole deployment.
+
+## Consequences
+
+Using the definition of levels established in this decision record throughout all SCS standards would allow readers to understand up to which level certain procedures or aspects of resources (e.g. volume types or a backend requiring redundancy) would protect their data and/or resource availability.
+
+### General Terms
+
+| Term               | Explanation                                                                                                                              |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| Disk               | A physical disk drive (e.g. HDD, SSD) in the infrastructure.                                                                             |
+| Host               | A physical machine in the infrastructure providing computational, storage and/or network connectivity capabilities.                      |
+| Cyber threat       | Attacks on the infrastructure through the means of electronic access.                                                                    |
+
+### Affected Resources
+
+#### IaaS Layer (OpenStack Resources)
+
```

Comment:
Affect resource table: we should add  from which level/class of failuresafeness, these resources are affected.